superset-frontend/cypress-base/cypress.config.ts

```diff
@@ -26,9 +26,9 @@ export default eyesPlugin(
     defaultCommandTimeout: 8000,
     numTestsKeptInMemory: 0,
     experimentalFetchPolyfill: true,
+    experimentalMemoryManagement: true,
     requestTimeout: 10000,
     video: false,
-    videoUploadOnPasses: false,
```

Comment:
Was this removal intentional? I don't see it mentioned in the docs or the description..

Reply:
Yeah, it was flagged by typescript that it no longer exists.

Reply:
Oh ok, that explains why it wasn't mentioned in the docs 😄